libs/blockscout-service-launcher/src/launcher/settings.rs

```diff
@@ -117,6 +119,30 @@ impl CorsSettings {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(try_from = "String")]
+pub struct BasePath(pub String);
```

Comment:
```suggestion
pub struct BasePath(String);
```
I think it's better to force parsing the string to prevent possible incorrect values